tests/tasks/assert_output_logs.yml

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Maybe give this file a better name that describes the intended outcome like `assert_output_in_stderr_without_warnings.yml`- what do you think?

Reply:
I tried to keep it short and clear, but this name is much more accurate, thanks!